pages/common/kubectl-run.md

```diff
@@ -0,0 +1,24 @@
+# kubectl run
+
+> Run pods in Kubernetes. Specifies pod generator to avoid deprecation error in some K8S versions.
+> More information: <https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#run>.
+
+- Start an nginx pod and expost port 80
+
+`kubectl run --generator=run-pod/v1 nginx --image=nginx --port 80`
+
+- Run an nginx pod, setting the TEST_VAR environment variable:
+
+`kubectl run --generator=run-pod/v1 nginx --image=nginx --env="TEST_VAR=testing"`
+
+- Show API calls that would be made to create an nginx container:
+
+`kubectl run --generator=run-pod/v1 nginx --image=nginx --dry-run`
+
+- Run a one-off Ubuntu pod interactively, never restart it, and remove it when it exits:
+
+`kubectl run --generator=run-pod/v1 -it temp-ubuntu --image=ubuntu:20.04 --restart=Never --rm -- /bin/bash`
+
+- Run a Ubuntu pod overriding the default command with echo and specifying custom arguments:
```

Comment:
```suggestion
- Spawn a Ubuntu pod, override the default command with 'echo' and specify custom arguments:
```

Reply:
Again, this is mostly fine as-is - but could probably do with a comma or two:
```suggestion
- Run an Ubuntu pod, overriding the default command with echo, and specifying custom arguments:
```